src/main/scala/mesosphere/marathon/util/RichEither.scala

```diff
@@ -0,0 +1,39 @@
+package mesosphere.marathon
+package util
+
+import scala.collection.generic.CanBuild
+
+object RichEither {
+  // Remove for Scala 2.12
+  implicit class RightBiased[A, B](val e: Either[A, B]) extends AnyVal {
+    def foreach[U](f: B => U): Unit = e.right.foreach(f)
+    def map[C](f: B => C): Either[A, C] = e.right.map(f)
+    def flatMap[C](f: B => Either[A, C]) = e.right.flatMap(f)
+  }
+
+  import scala.language.higherKinds
+  /**
+    * If any lefts are in the sequence, then returns a sequence of all lefts
```

Comment:
I would rather the document not explain what it is technically doing... but what it is used for and why it exists.... when would a developer choose this over Either? etc.

Reply:
I wrote this. The behavior is similar to that of `Future.sequence`. Looks like it may not be referenced at the time, so perhaps the function can just be removed.